File: BATCH_21/js and python/c3_functions.py
# defining a function
def greet():
    print("Hello 1")
    print("Hello 2")
    print("Hello 3")
    print("Hello 4")
    print("Hello 5")
    print("Hello 6")


# greet()

# def fullname(fname, lname):
#     result = fname + " " + lname
#     return result

fullname = lambda fname, lname: fname + " " + lname

fname = input("Your first name\n=> ")
lname = input("Your last name\n=> ")

full = fullname(fname, lname)

print(full)

File: BATCH_21/js and python/c3_functions.js
/*
// function declaration
function greet() {
  console.log("Hello 1");
  console.log("Hello 2");
  console.log("Hello 3");
  console.log("Hello 4");
  console.log("Hello 5");
  console.log("Hello 6");
}

// arrow function
const greet = () => {
  console.log("Hello 1");
  console.log("Hello 2");
  console.log("Hello 3");
  console.log("Hello 4");
  console.log("Hello 5");
  console.log("Hello 6");
}

// function calling / invoking
greet();
*/

// function parameters
// function expression
// function fullname(fname, lname) {
//   const result = fname + " " + lname;
//   return result;
// }

// const fullname = (fname, lname) => {
//   const result = fname + " " + lname;
//   return result;
// };

const fullname = (fname, lname) => fname + " " + lname;

const full = fullname("Vijay", "Kumar");
const full2 = fullname("Ajay", "Kumar");

console.log(full);
console.log(full2);
